apps/ledger-live-desktop/tests/models/ReceiveModal.ts

```diff
@@ -5,12 +5,22 @@ export class ReceiveModal extends Modal {
   readonly page: Page;
   readonly skipDeviceButton: Locator;
   readonly verifyMyAddressButton: Locator;
+  readonly verifyAddressOnDevice: Locator;
```

Comment:
could you please make sure that all your newly added variables across the project contain in the name type of the element? as in "submitButton" instead of "submit", etc